app/communication/models/banner.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+
+from django.db import models
+from django.utils import timezone
+
+from app.common.enums import AdminGroup
+from app.common.permissions import BasePermissionModel
+from app.util.models import BaseModel, OptionalImage
+from app.util.utils import now
+
+
+class Banner(BaseModel, OptionalImage, BasePermissionModel):
+    write_access = AdminGroup.admin()
+    read_access = AdminGroup.admin()
+    id = models.UUIDField(
+        auto_created=True, primary_key=True, default=uuid.uuid4, serialize=False,
+    )
+    title = models.CharField(max_length=100)
+    description = models.TextField()
+    visible_from = models.DateTimeField(default=timezone.now)
+    visible_until = models.DateTimeField(blank=True, null=True)
```

Comment:
Burde vi hatt en constraint som hindrer at flere bannere overlapper hverandre?

Reply:
Tenker det kan være gunstig at det er mulig med flere enn et banner. For eksempel om det først er informasjon om en oppdatering med nye features, men så er det en til som blir spontant laget for å si at man har fikset en bug. Eller er det noe annet som bannere skal brukes for? Men jeg enig at man ikke skal ha mulighet til 100 bannere samtidig.